docs/source/_static/msgspec-logo-dark.svg

```diff
@@ -1,4 +1,4 @@
-<svg version="1.1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 169 140.4330930481983" width="338" height="280.8661860963966" filter="invert(93%) hue-rotate(180deg)">
+<svg version="1.1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 169 140.4330930481983" width="338" height="280.8661860963966">
```

Comment:
I don't do web things at all. Is there a reason you modified the svgs here this way?

Also to me the `*-dark.svg` is for dark backgrounds and `*-light.svg` is for light backgrounds, is there a reason you reversed this? These logos are used in the docs as well, so any changes like this would need to be propagated throughout the docs build too.

Reply:
im not sure what happened to that change, but on the light vs dark i think i got them confused and flipped them 😬